src/use_case_classification.Rmd

```diff
@@ -0,0 +1,152 @@
+# Classification: Predicting Survival on the Titanic
+
+One popular example regarding a classification task is the Titanic showcase. We have different passenger information - like name, age or fare - available with the aim to predict which kind of people would have survived the Titanic sinking.
+
+Therefore we load the titanic dataset and other packages that are needed for this use case.
+
+```{r, warning=FALSE, message=FALSE}
+library(titanic)
+library(mlr)
+library(BBmisc)
+```
+
+
+```{r}
+data = titanic_train
+head(data)
+```
+
+Our aim - as mentioned before - is to predict which kind of people would have survived.
+
+Therefore we will work off the following steps:
+
+* preprocessing, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/preproc/index.html) and [here](http://mlr-org.github.io/mlr-tutorial/devel/html/impute/index.html)
```

Comment:
I overlooked this during my first review. If you want to link to other tutorial pages please use e.g. `[here](preproc.md)`.